code/online_challenges/src/hackerrank/full_counting_sort/full_counting_sort.cpp

```diff
@@ -0,0 +1,43 @@
+#include <string>
+#include <iostream>
+
+int main() 
+{
+    long int n;
+    
+    std::cin >> n;
+    
+    std::__cxx11::string ar[n];
```

Comment:
Use a regular `std::string`. Also, C++ does not support VLAs. Use a `std::vector` instead.

Reply:
This is still not resolved.